src/Pos/Ssc/GodTossing/GState.hs

```diff
@@ -81,15 +93,66 @@ instance SscGStateClass SscGodTossing where
         view gsOpenings <*>
         view gsShares
 
-loadGlobalState :: (MonadDB m, WithLogger m) => m GtGlobalState
+loadGlobalState
+    :: forall m .
+    ( MonadDB m
+    , WithLogger m
+    , HasContext LrcContext m
+    ) => m GtGlobalState
 loadGlobalState = do
     logDebug "Loading SSC global state"
-    gs <- DB.getGtGlobalState
+    gs <- DB.getGtGlobalState `catch` fallbackLoad
     gs <$ logInfo (sformat ("Loaded GodTossing state: " %build) gs)
+  where
+    fallbackLoad :: SomeException -> m GtGlobalState
+    fallbackLoad _ = do
+        logInfo $ "GtGlobalState dump isn't found, load blocks (workaround)"
+        loadGlobalStateOld
```

Comment:
Do you tell about the workaround or about the approach at all?
If you about approach at all: I want to do it atomically and consistently (with other code)
If you about workaround: I think, it doesn't matter